modules/server-node/prisma-sqlite/schema.prisma

```diff
@@ -208,19 +239,15 @@ model OnchainTransaction {
   logs              String?
   cumulativeGasUsed String?
   byzantium         Boolean?
-
-  // Channel fields
-  status         String  @default("submitted") // no enums
-  reason         String // no enums
-  error          String?
```

Comment:
Should error not be present? Is that relegated to the `OnchainTransaction` parent?